tests/test_losca_metro_transit.py

```
@@ -0,0 +1,86 @@
+
+def test_location():
+    assert parsed_items[0]["location"] == {
+        "name": "Board of Directors - Regular Board Meeting",
+        'address': 'One Gateway Plaza, Los Angeles, CA 90012, \r\n'
+        '3rd Floor, Metro Board Room Watch online:  '
+        'https://boardagendas.metro.net \r\n'
+        'Listen by phone: Dial 202-735-3323 and enter Access Code:\r\n'
+        '5647249# (English) or 7292892# (Español) \r\n'
+        '\r\n'
+        'To give written or live public comment, please see the top of page 4',
+        'name': 'Board of Directors - Regular Board Meeting',
```

Ruff

Dictionary key literal `'name'` repeated

Remove repeated key literal `'name'`

(F601)